index_V1.html

```diff
@@ -14,9 +14,8 @@
     </div>
     <ul class="menu">
 		<li><a href="index.html">Home</a></li>
-
-		<li><a href="index2.html">The Tour</a></li>
-		<li><a href="contact.html">Contact Form</a></li>
+		<li><a href="index2.html">page 2</a></li>
+		<li><a href="contact.html">page 3</a></li>

```

Comment:
23/06/2021. I change the nav link titles from index2 and contact to page 2 and 3.